### ISSUE TITLE
Error: 504: Gateway Timeout from github

### ISSUE THREAD

**Issue by AdrianSkierniewski** [webpro/release-it#50]
Hi,

I was trying to create github release using release-it but after running release-it without any errors, github release was missing.  I tried to find what is going on and I manage to log error response from `githubClient.releases.createRelease`. I was getting 504 response all the time but without any warnings from release-it. I double check my token and I mange to send a request using Postman REST client. I don't know if this is related but there is new version of github module 1.1.0 and old one is deprecated. I spend some time trying to log what was send to github but I didn't find anything unusually.

```{ [Error: 504: Gateway Timeout]
  defaultMessage: 'Gateway Timeout',
  message: '504: Gateway Timeout',
  code: '504' }```

**Comment by AdrianSkierniewski** [comment]
I solved this issue by increasing timeout to 50000

```
     _githubClient = new GitHubApi({
            version: '3.0.0',
            debug: config.isDebug(),
            protocol: 'https',
            host: 'api.github.com',
            timeout: 50000,
            headers: {
                'user-agent': 'webpro/release-it'
            }
        });
```

**Comment by webpro** [comment]
Hi @AdrianSkierniewski, thanks for the feedback. I want to upgrade to the latest version of the `github` package. But I don't really like to increase the timeout to 50 seconds for everyone.

Maybe we can negotiate, and make it 10 seconds? :-)

**Comment by AdrianSkierniewski** [comment]
sure :)

**Comment by webpro** [comment]
I've just released this as [v2.4.3](https://github.com/webpro/release-it/releases/tag/2.4.3).
